assets/js/theme/common/faceted-search.js

```diff
@@ -253,7 +254,7 @@ class FacetedSearch {
             minPriceSelector: this.options.priceRangeMinPriceSelector,
         };
 
-        Validators.setMinMaxPriceValidation(validator, selectors);
+        Validators.setMinMaxPriceValidation(validator, selectors, this.options.validationErrorMessages);
```

Comment:
Actually, it's not empty object. It's empty within defaultOptions. There are 3 instances of facetedSearch. In Category, Brand and Search pages. During initiating they take 3rd parameter - options, where will be validationErrorMessages object with provided texts (check Category.js lines 61-67).
this.options Object itself is created by combining defaultOptions and options and eventually will contain validationErrorMessages.

Reply:
Could we use default func param instead?

Reply:
you're suggesting to use default parameter in setMinMaxPriceValidation definition, is it correct? If so we can't do this since Validators object doesn't have access to translations(validation errors). Instead, we throw them from above to FacetedSearch class instances and only after that can provide to setMinMaxPriceValidation call.

Reply:
I believe default param will free us from defining empty obj in defaultOptions obj